.github/workflows/run-tests.yml

```diff
@@ -82,9 +82,20 @@ jobs:
         run: |
           npm install
 
+      # Will also run unit tests, but that's almost free.
+      # Otherwise, we need to set up a separate job,
+      # which will also run the integration tests for the second time,
+      # and that's more time-consuming.
       - name: Run integration tests
         run: |
-          npm run test:integration
+          npm t -- --coverage
```

Comment:
Let's add an instruction to `CONTRIBUTING.md` about collecting the coverage locally.